src/main/java/com/epam/preprod/sirenko/command/ExitProgramCommand.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.preprod.sirenko.command;
+
+import com.epam.preprod.sirenko.entity.Product;
+
+import java.io.IOException;
+import java.net.Socket;
+import java.util.List;
+
+public class ExitProgramCommand extends Command {
+	//TODO
+	@Override
+	public List<Product> execute() {
+		try (Socket socket = new Socket()) {
```

Comment:
Why do we need socket here? Currently we just doing console interface for user

Reply:
this method is totally wrong, I'll fix it